gui/src/store/modules/user-login.ts

```diff
@@ -122,6 +133,26 @@ export default class UserLogin extends VuexModule {
     }
   }
 
+  @Action({ rawError: true })
+  public async getUnsupportedFeaturesAction() {
+    try {
+      const data = this.context.getters["getUnsupportedFeatures"];
+      if (!data.unsupported_features.length) {
+        const features = await Api.getAll(apiRegister.unsupported_features);
+        if (features && features.data) {
```

Comment:
You can flatten the features array to an object. Ex:
{
capacity: true,
hctl_node: true,
health: true
}
This will make it easier to access the feature.

Reply:
Updated logic as discussed.